app/src/main/AndroidManifest.xml

```diff
@@ -257,46 +224,18 @@
                 <data android:scheme="antennapod-subscribe"/>
             </intent-filter>
 
-            <!-- Support for subscribeonandroid.com URLS -->
-            <intent-filter>
-                <action android:name="android.intent.action.VIEW" />
-
-                <category android:name="android.intent.category.DEFAULT" />
-                <category android:name="android.intent.category.BROWSABLE" />
-
-                <data android:pathPattern="/.*\\..*/.*" />
-                <data android:host="subscribeonandroid.com" />
-                <data android:host="www.subscribeonandroid.com" />
-                <data android:host="*subscribeonandroid.com" />
-                <data android:scheme="http" />
-                <data android:scheme="https" />
-            </intent-filter>
-
             <intent-filter>
```

Comment:
Shouldn't we make this an [Android App link](https://developer.android.com/training/app-links#android-app-links) by setting `android:autoVerify="true"` for our own domain?

Benefit: "when the user clicks on an Android App Link, your app opens immediately if it's installed—the disambiguation dialog doesn't appear."